src/main/java/seedu/address/storage/JsonSerializableAddressBook.java

```diff
@@ -57,4 +57,14 @@ public AddressBook toModelType() throws IllegalValueException {
         return addressBook;
     }
 
+    @Override
+    public String toString() {
+        StringBuilder builder = new StringBuilder();
+        builder.append("JsonSerializableAddressBook:\n");
+        for (JsonAdaptedPerson jAP : persons) {
+            builder.append("[" + jAP + "]\n");
```

Comment:
Is there any reason to use that naming style for the `JsonAdaptedPerson` variable `jAP`?
```suggestion
        for (JsonAdaptedPerson person : persons) {
            builder.append("[" + person + "]\n");
```